src/OrchardCore/OrchardCore/Environment/Shell/Builders/ShellContainerFactory.cs

```diff
@@ -101,8 +101,13 @@ public IServiceProvider CreateContainer(ShellSettings settings, ShellBlueprint b
                     "Configure",
                     BindingFlags.Public | BindingFlags.Instance);
 
-                var orderProperty = rawStartup.GetProperty("Order");
-                var configureOrderProperty = rawStartup.GetProperty("ConfigureOrder");
+                var orderProperty = rawStartup.GetProperty(
```

Comment:
Could we use ``nameof` instead of magic string

```csharp
var orderProperty = rawStartup.GetProperty(nameof(Order), BindingFlags.Public | BindingFlags.Instance);

var configureOrderProperty = rawStartup.GetProperty(nameof(ConfigureOrder), BindingFlags.Public | BindingFlags.Instance);
```